### PR TITLE
Update Node version in Broker Dockerfile

### PR DESCRIPTION
**Pull request by anoopjb**
Update Node version to 14.15 LTS
- update in `dockerfile` of broker
- update in Travis configuration file

Feature: HCPCFS-2784

|Dockerfile|Current Version|Latest Stable Version|LTS|
|---|---|---|---|
|[broker](https://github.com/cloudfoundry-incubator/service-fabrik-broker/blob/master/broker/Dockerfile#L2) |node:12.18-alpine3.12|node:15.5.1|14.15.4|

[Node release schedule](https://github.com/nodejs/Release#release-schedule)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -97,7 +97,7 @@ matrix:
         - popd
     - language: node_js
       node_js:
-        - "12.18"
+        - "15.5"
```
Comment (anoopjb) [inline .travis.yml:100]:
```suggestion
        - "14.15"
```

File: .travis.yml [hunk 2]
```diff
@@ -107,7 +107,7 @@ matrix:
...
```
Comment (anoopjb) [inline .travis.yml:110]:
```suggestion
        - "14.15"
```

File: .travis.yml [hunk 3]
```diff
@@ -118,7 +118,7 @@ matrix:
...
```
Comment (anoopjb) [inline .travis.yml:121]:
```suggestion
        - "14.15"
```

File: broker/Dockerfile [hunk 4]
```diff
@@ -1,5 +1,5 @@
 # FROM node:12.18-alpine3.12 also works here for a smaller image
-FROM node:12.18-alpine3.12
+FROM node:15.5-alpine3.12
```
Comment (anoopjb) [inline broker/Dockerfile:2]:
```suggestion
FROM node:14.15-alpine3.12
```

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n is succeeded.

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n-2 is succeeded.

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n-1 is succeeded.

**Comment by vinaybheri** [comment]
Validation on k8s cluster: n is succeeded.

**Review by subhankarc — APPROVED** [review]

**Pull request merged**